python/tvm/relay/op/transform.py

```diff
@@ -42,12 +42,32 @@ def transpose(data, axes=None):
     Returns
     -------
     result : relay.Expr
-        The reshaped result.
+        The transposed result.
     """
     axes = axes or []
     return _make.transpose(data, list(axes))
 
 
+def squeeze(data, axes=None):
+    """Squeeze axes in the array.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data to the operator.
+
+    axes : None or List[int]
+        Axes to remove, remove all 1 if not specified, or it is [].
+
+    Returns
+    -------
+    result : relay.Expr
+        The squeezed result.
+    """
+    axes = axes or []
+    return _make.squeeze(data, list(axes))
```

Comment:
I've seen other code wrap lists in `tvm.convert`. Is that necessary here or unnecessary there?